Maps_Enum_HW/homework3/task2/Service.java

```diff
@@ -0,0 +1,35 @@
+package homework3.task2;
+
+public class Service {
+
+    public void traverseAndPrintAllMonthNames() {
+        System.out.println("Months of the year are:");
+        for (Month month : Month.values()) {
+            System.out.println(month);
+        }
+    }
+
+    public String getDaysInMonth(Month month, int year) {
+        if (month == Month.FEBRUARY && year % 4 == 0) {
```

Comment:
Checking for a leap year is not that straightforward https://www.timeanddate.com/date/leapyear.html , 
so you may want to use a library method https://www.baeldung.com/java-leap-year